src/bundle/Command/MultiprocessComand.php

```diff
@@ -0,0 +1,357 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\FieldTypeRichText\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\PhpExecutableFinder;
+use Symfony\Component\Process\Process;
+
+abstract class MultiprocessComand extends Command
+{
+    /** @var \Ibexa\Contracts\Core\Repository\PermissionResolver */
+    protected $permissionResolver;
```

Comment:
```suggestion
    protected PermissionResolver $permissionResolver;
```

You can use explicit type declaration in php 7.4 (which is min. supported version for 4.3 branch). Please apply this comment to all occurences.

Reply:
Fixed in 8264f2d52fb01eaa02784f31a9e28f02e13c6f3e, except for `private $processes` which is of type `Process[]`. That one should be left as-is, right ?

Reply:
Fixed in 8264f2d52fb01eaa02784f31a9e28f02e13c6f3e